index.html

```diff
@@ -26,13 +26,17 @@
         background: #f3f3f3;
       }
 
-      h1 {
-        font-size: 2em;
-      }
-
       h1, h2, h3, h4, h5, h6 {
         font-family: "Rubik", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
-        font-weight: 500;
+        font-weight: 400;
+        line-height: 1;
```

Comment:
I think the default `line-height: 1.5;` is more appropriate.
otherwise when they wrap it'll look like this

![image](https://user-images.githubusercontent.com/9994172/46985400-0230ec80-d136-11e8-9417-278ce1214b36.png)